src/main/java/net/coderbot/terrestria/init/TerrestriaBiomes.java

```diff
@@ -409,7 +426,11 @@ public static void init() {
 		OverworldBiomes.addContinentalBiome(TEMPERATE_RAINFOREST, OverworldClimate.COOL, 1.0);
 		OverworldBiomes.addContinentalBiome(SNOWY_RAINFOREST, OverworldClimate.SNOWY, 2.0);
 		OverworldBiomes.addContinentalBiome(ALPINE, OverworldClimate.SNOWY, 1.0);
-
+		
+		OverworldBiomes.addHillsBiome(ALPINE, ALPINE, 1);
```

Comment:
This is to make the alps less common as hills

Reply:
Got it, makes sense.